src/filters.ts

```diff
@@ -244,4 +244,18 @@ angular.module('3ema.filters', [])
     };
 }])
 
+/**
+ * Convert ID-Array to (Display-)Name-String, separated by ','
+ */
+    .filter('idToName', ['WebClientService', function (webClientService: threema.WebClientService) {
```

Comment:
The `.filter` call should not be indented, to match the rest of the file.

Also, the function should probably be named `idsToNames`, since it accepts multiple ids.